staff-user-documentation/README.md

```diff
@@ -0,0 +1,396 @@
+
+![Search Interface](images/license_search_tab.png)
+
+#### Search Criteria
+You can search using the following parameters:
+- **State/Jurisdiction**: Select from dropdown menu. This will filter the results to only display practitioners that have a license or privilege in the specified jurisdiction.
+- **Practitioner Name**: Enter full first and last name
+- **Combined Search**: Use both state and name criteria
+
+⚠️ **Note**: Partial name searches are not currently supported. You must enter the complete first and last name.
+
+#### Performing a Search
+1. Enter your search criteria
+2. Click **"Search"**
+3. Review results on the License Listing page
+4. Click any row to view detailed practitioner information
+
+![Search Results](images/license_list_view.png)
+
+### Practitioner Details Page
+
+The practitioner details page displays comprehensive license and privilege information for individual practitioners.
+
+![Practitioner Details](images/practitioner_details_page.png)
+
+#### Available Information:
+- Licenses and their current status
+- Privileges and their current statis
+- Privilege history and timeline
+
+#### Viewing Privilege History
+Click **"View Details"** on any privilege card to access:
+- Complete privilege timeline, including status changes such as deactivation and encumbrance.
+- Status change history
+
+![Privilege Timeline](images/privilege_summary.png)
+
+---
+
+## Data Upload
+
+### License Information Upload
+**This feature is available to staff users with write permissions only.**
+
+#### Uploading License Data
+1. Select **"Upload Data"** from the left navigation panel
+![Data Upload](images/license_upload.png)
+2. Click **"Select File"** to select your CSV document
+3. Ensure your license data in the CSV file includes all required license data fields as specified in the [License Data Schema Documentation](../backend/compact-connect/docs/README.md).
+4. Click **"Submit"** to process the upload
+
+#### Upload Validation
+The system will validate your data and provide weekly email feedback on:
+- Successful imports
+- Data formatting errors
+- Missing required fields
+
+These email notifications will be sent to whichever email addresses have been set by the state admin for your respective state's Operations notification recipients.
+
+---
+
+## Automated License Data Upload for IT Departments
+
+### Overview
+State IT departments can set up automated license data uploads to CompactConnect through the API, eliminating the need for manual CSV uploads and ensuring timely data synchronization between state licensing systems and CompactConnect. For more information about this setup, see the following document:
+
+**[Automated License Data Upload Onboarding Instructions](../backend/compact-connect/app_clients/it_staff_onboarding_instructions/README.md)**
+
+---
+
+## Privilege and License Management
+
+### Deactivating Privileges
+**This feature is available to Compact Administrators only.**
```

Comment:
We should probably explain a bit here about the intended use - some positive examples to clarify _why_ these are different from encumbrances.